packages/database/src/migrations/20210112015125-AddHeatmapContactsConfirmedSuspectedCasesSCovidSamoa-modifies-data.js

```diff
@@ -35,20 +35,19 @@ async function deleteMapOverlayInMapOverlayGroupRelation(db, childId) {
 }
 
 const mapOverlay = {
-  id: 'COVID_WS_Address_Contacts_Of_Suspected_And_Confirmed_Cases_WS',
-  name: 'Home village of contacts of suspected/confirmed cases',
-  userGroup: 'Donor',
+  id: 'COVID_WS_Address_Contacts_Of_Suspected_And_Confirmed_Cases',
+  name: 'WIP Home village of contacts of suspected/confirmed cases',
```

Comment:
Just confirming that `WIP` is desirable in the name?

Reply:
Ack! No. Fixing.